python_anvil/api_resources/requests.py

```diff
@@ -161,3 +161,29 @@ class PlainRequest(BaseAnvilHttpRequest):
 
     def get_url(self):
         return f"{self.API_HOST}/{self.API_BASE}"
+
+
+class FullyQualifiedRequest(BaseAnvilHttpRequest):
+    """A request class that validates URLs point to Anvil domains."""
+
+    VALID_HOSTS = [
+        "https://app.useanvil.com",
+        # Future Anvil specific URLs
+    ]
+
+    def get_url(self):
+        return ""  # Not used since we expect full URLs
```

Comment:
need to define this? the base of this func throws an error, prob can keep if we don't want people to use this

Reply:
this is intentionally defined as an empty string, otherwise request_rest tries to append whatever URL is passed in to whatever is returned in get_url

Reply:
if that's the case then yeah def, but doesn't this function get inherited from [here](https://github.com/anvilco/python-anvil/blob/28e4d3f8ddf73e1f14979debef857a717979792d/python_anvil/api_resources/requests.py#L62)? I'm not a python guy so idk how inheritance works here

Reply:
it does get inherited from [there](https://github.com/anvilco/python-anvil/blob/28e4d3f8ddf73e1f14979debef857a717979792d/python_anvil/api_resources/requests.py#L62) but we are overriding it here by returning an empty string